avocado/core/test.py

```diff
@@ -469,6 +469,9 @@ def _run_avocado(self):
         except exceptions.TestSkipError as details:
             stacktrace.log_exc_info(sys.exc_info(), logger='avocado.test')
             raise exceptions.TestSkipError(details)
+        except exceptions.TestSkip as details:
```

Comment:
Also, 'Error' in the name fells inaccurate, since an `SKIP` is considered a 'good' test status.